kogito-kafka-quickstart-springboot/README.md

```diff
@@ -71,8 +129,21 @@ To make use of this application it is as simple as putting a message on `travell
 
 this will then trigger the successful processing of the traveller and put another message on `processedtravellers` topic.
 
+* To examine ProcessedTravellers topic and verify it was auto sent
+
+```
+kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers --from-beginning
+```
+
 To take the other path of the process put following message on `travellers` topic
 
+* Send Message to Topic
+```
+kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
+```
+
+Content
+
```

Comment:
This content does not work, it should be full, like the other message sent:

```
{
  "specversion": "0.3",
  "id": "21627e26-31eb-43e7-8343-92a696fd96b1",
  "source": "",
  "type": "TravellersMessageDataEvent_3",
  "time": "2019-10-01T12:02:23.812262+02:00[Europe/Warsaw]",
  "data": { 
	"firstName" : "John", 
	"lastName" : "Doe", 
	"email" : "john.doe@example.com", 
	"nationality" : "American"
	}
}
```

Reply:
and please put the json as a one-line, as the user may want to copy/paste it

Reply:
fixed @radtriste